source/sass/components/article-page.scss

```diff
@@ -312,6 +312,41 @@
     @include set-text-size(16px, 24px);
     font-family: $font-family-sans-serif;
     font-weight: normal;
+
+    .rich-text {
+      * {
+        color: $gray-80;
+        font-size: 16px;
+      }
+
+      p {
+        max-width: initial;
+      }
+
+      b {
+        font-weight: bold;
+      }
+
+      a {
+        color: $dark-blue;
```

Comment:
@sabrinang what style should links on article page content be? Do they appear with underline? How about hover state style?

Reply:
@mmmavis I'm good with same anchor link styles we have in style guide already _(so underline on hover)_

Reply:
I believe these are the same in both tables already.